src/rgw/rgw_kafka.cc

```diff
@@ -611,6 +675,17 @@ class Manager {
     }
     return STATUS_QUEUE_FULL;
   }
+
+  int check_broker(const std::string& conn_name,
+    const std::string& topic) {
```

Comment:
we create the topic as part of the connection creation, if the creation failed this will be reflected in the status.
added it as a parameter for the generic SAL interface (might be needed in other cases).
but i can remove for now